CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -415,15 +418,34 @@ local function Create3DMenuItem()
 	}
 
 	local isActive = false
+	local isModal = true
```

Comment:
This is a separate concept from panel modal-ness. Should we perhaps name it "isExclusive" or something?